From fd574adf6e0a3f3acd7ee4563e81ab941c3e0c6a Mon Sep 17 00:00:00 2001
From: Rahul T R <r-ravikumar@ti.com>
Date: Fri, 21 Oct 2022 16:34:05 +0530
Subject: [PATCH 2/3] arm64: dts: ti: k3-j721s2: Add overlay for edge ai

Add overlay for RTOS memory map for edge ai with
DSS and other pheriperals enabled

Signed-off-by: Rahul T R <r-ravikumar@ti.com>
---
 arch/arm64/boot/dts/ti/Makefile                  | 1 +
 arch/arm64/boot/dts/ti/k3-j721s2-edgeai-apps.dts | 8 ++++++++
 2 files changed, 9 insertions(+)
 create mode 100644 arch/arm64/boot/dts/ti/k3-j721s2-edgeai-apps.dts

diff --git a/arch/arm64/boot/dts/ti/Makefile b/arch/arm64/boot/dts/ti/Makefile
index 3cc263dafa..de32c2c38f 100644
--- a/arch/arm64/boot/dts/ti/Makefile
+++ b/arch/arm64/boot/dts/ti/Makefile
@@ -54,6 +54,7 @@ dtb-$(CONFIG_ARCH_K3) += k3-j721s2-cpb-csi2-ov5640.dtbo
 dtb-$(CONFIG_ARCH_K3) += k3-j721s2-fpdlink-cpb-fusion.dtbo
 dtb-$(CONFIG_ARCH_K3) += k3-j721s2-common-proc-board-uarts.dtbo
 dtb-$(CONFIG_ARCH_K3) += k3-j721s2-vision-apps.dtbo
+dtb-$(CONFIG_ARCH_K3) += k3-j721s2-edgeai-apps.dtbo
 
 dtb-$(CONFIG_ARCH_K3) += k3-am69-sk.dtb
 dtb-$(CONFIG_ARCH_K3) += k3-am69-sk-csi2-ov5640.dtbo
diff --git a/arch/arm64/boot/dts/ti/k3-j721s2-edgeai-apps.dts b/arch/arm64/boot/dts/ti/k3-j721s2-edgeai-apps.dts
new file mode 100644
index 0000000000..5c2add991e
--- /dev/null
+++ b/arch/arm64/boot/dts/ti/k3-j721s2-edgeai-apps.dts
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * vision-apps: device-tree overlay
+ *
+ * Copyright (C) 2019 - 2021 Texas Instruments Incorporated - http://www.ti.com/
+ */
+
+#include "k3-j721s2-rtos-memory-map.dtsi"
-- 
2.17.1

